command/server/command.go

```diff
@@ -9,28 +9,193 @@ import (
+	return s.flagOptional
+}
+
 // Command is the command to start the sever
 type Command struct {
 	UI cli.Ui
+
+	flagMap map[string]types.FlagDescriptor
+}
+
+// DefineFlags defines the command flags
+func (c *Command) DefineFlags() {
+	if c.flagMap == nil {
+		// Flag map not initialized
+		c.flagMap = make(map[string]types.FlagDescriptor)
+	}
+
+	if len(c.flagMap) > 0 {
+		// No need to redefine the flags again
+		return
+	}
+
+	c.flagMap["log-level"] = ServerFlagDescriptor{
+		description: fmt.Sprintf("Sets the log level for console output. Default: %s", defaultConfig().LogLevel),
+		arguments: []string{
+			"LOG_LEVEL",
+		},
+		flagOptional: true,
+	}
+
+	c.flagMap["seal"] = ServerFlagDescriptor{
+		description: "Sets the flag indicating that the client should seal blocks. Default false",
+		arguments: []string{
+			"SHOULD_SEAL",
+		},
+		flagOptional: true,
+	}
+
+	c.flagMap["config"] = ServerFlagDescriptor{
+		description: "Specifies the path to the CLI config. Supports .json and .hcl",
+		arguments: []string{
+			"CLI_CONFIG_PATH",
+		},
+		flagOptional: true,
+	}
+
+	c.flagMap["chain"] = ServerFlagDescriptor{
+		description: fmt.Sprintf("Specifies the genesis file used for starting the chain. Default %s", defaultConfig().Chain),
+		arguments: []string{
+			"GENESIS_FILE",
+		},
+		flagOptional: true,
+	}
+
+	c.flagMap["data-dir"] = ServerFlagDescriptor{
+		description: fmt.Sprintf("Specifies the data directory used for storing Polygon SDK client data. Default %s", defaultConfig().DataDir),
+		arguments: []string{
+			"DATA_DIRECTORY",
+		},
+		flagOptional: true,
+	}
+
+	c.flagMap["grpc"] = ServerFlagDescriptor{
+		description: fmt.Sprintf("Sets the address and port for the gRPC service (address:port). Default address: 127.0.0.1:%d", minimal.DefaultGRPCPort),
+		arguments: []string{
+			"GRPC_ADDRESS",
+		},
+		flagOptional: true,
+	}
+
+	c.flagMap["jsonrpc"] = ServerFlagDescriptor{
+		description: fmt.Sprintf("Sets the address and port for the JSON-RPC service (address:port). Default address: 127.0.0.1:%d", minimal.DefaultJSONRPCPort),
+		arguments: []string{
+			"JSONRPC_ADDRESS",
+		},
+		flagOptional: true,
+	}
+
+	c.flagMap["libp2p"] = ServerFlagDescriptor{
+		description: fmt.Sprintf("Sets the address and port for the libp2p service (address:port). Default address: 127.0.0.1:%d", network.DefaultLibp2pPort),
+		arguments: []string{
+			"LIBP2P_ADDRESS",
+		},
+		flagOptional: true,
+	}
+
+	c.flagMap["join"] = ServerFlagDescriptor{
+		description: "Specifies the address of the peer that should be joined",
+		arguments: []string{
+			"JOIN_ADDRESS",
+		},
+		flagOptional: true,
+	}
+
+	c.flagMap["nat"] = ServerFlagDescriptor{
+		description: "Sets the the external IP address without the port, as it can be seen by peers",
+		arguments: []string{
+			"NAT_ADDRESS",
+		},
+		flagOptional: true,
+	}
+
+	c.flagMap["no-discover"] = ServerFlagDescriptor{
+		description: "Sets the client's discoverability for other peers. Default false",
```

Comment:
By this description, I would suppose that `true` would set the client as discoverable by other peers.
```suggestion
		description: "Prevents the client from discovering other peers. Default false",
```